plugins/chat/app/services/chat/search_chatable.rb

```diff
@@ -9,6 +9,8 @@ module Chat
   class SearchChatable
     include Service::Base
 
+    SEARCH_RESULT_LIMIT ||= 10
```

Comment:
Added this constant to ensure all "search queries" return a maximum of 10 results.